### PR TITLE
Fix account.key by ensuring that the key is properly generated

### PR DESCRIPTION
**Pull request by ellmetha**

### CODE REVIEW

File: tasks/system.yml [hunk 1]
```diff
@@ -28,10 +28,18 @@
   become: yes
   become_user: "{{ acme_user }}"
 
-- name: Ensure that we have a letsencrypt account key
-  shell: "openssl genrsa 4096 > {{ acme_ssl_base_folder }}/account.key"
-  args:
-    creates: "{{ acme_ssl_base_folder }}/account.key"
+- name: Check if the account.key file already exists
+  stat: path="/root/account.key"
```
Comment (hsoft) [inline tasks/system.yml]: Why not `{{ acme_ssl_base_folder }}/account.key`? It's going to break the call to `acme_tiny`.

File: tasks/system.yml [hunk 2]
```diff
@@ -28,10 +28,18 @@
...
+  register: account_key_file
+
+- name: Generates a letsencrypt account key if necessary
+  command: "openssl genrsa 4096"
+  register: account_key
+  when: account_key_file.stat.exists == False
```
Comment (hsoft) [inline tasks/system.yml]: `not account_key_file.stat.exists` doesn't work?

File: tasks/system.yml (hunk 1 above)
Comment (ellmetha) [inline tasks/system.yml]: Exact. Sorry for that. ;-)

**Comment by hsoft** [comment]
Thanks.

**Pull request merged**